jscomp/others/node_buffer.ml

```diff
@@ -26,208 +26,696 @@
 
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_equals_otherbuffer> Node.js API docs
+    @since Node.js v0.11.13
+  *)
+external equals : t -> t -> bool = "equals" [@@bs.send]
+
+(** Fills [buf] with the specified [value]. If the [offset] and [end_] are not given, 
+    the entire [buf] will be filled.
+
+    @param buf
+    @param value The value with which to fill [buf].
+    @param offset Number of bytes to skip before starting to fill [buf]. Default: [0].
+    @param end_ Where to stop filling buf (not inclusive). Default: [length buf].
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_fill_value_offset_end_encoding> Node.js API docs
+    @since Node.js v0.5.0
+ *)
 external fill :
   t ->
-  ([ `String of string | `Integer of int | `Buffer of t ] [@bs.unwrap]) ->
+  ([ `Integer of int | `Buffer of t | `Uint8Array of Js_typed_array2.Uint8Array.t ] [@bs.unwrap]) ->
+  ?offset:int ->
+  ?end_:int ->
+  unit -> t = "fill"
+[@@bs.send]
+
+(** Fills [buf] with the specified [value]. If the [offset] and [end_] are not given, 
+    the entire [buf] will be filled.
+
+    @param buf
+    @param value The value with which to fill [buf].
+    @param offset Number of bytes to skip before starting to fill [buf]. Default: [0].
+    @param end_ Where to stop filling buf (not inclusive). Default: [length buf].
+    @param encoding The encoding for value. Default: [`utf8].
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_fill_value_offset_end_encoding> Node.js API docs
+    @since Node.js v0.5.0
+ *)
+external fillWithString :
+  t ->
+  string ->
   ?offset:int ->
   ?end_:int ->
   ?encoding: ([ `ascii  | `utf8  | `utf16le  | `usc2  | `base64  | `latin1 | `binary | `hex ] [@bs.string]) ->
   unit -> t = "fill"
 [@@bs.send]
 
+(** Tests, if [buf] has provided [value] looking from [byteOffset].
+
+    @param buf
+    @param value What to search for.
+    @param byteOffset Where to begin searching in [buf]. If negative, then offset is calculated from the end of [buf]. Default: [0].
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_includes_value_byteoffset_encoding> Node.js API docs
+    @since Node.js v5.3.0
+  *)
 external includes :
+  t -> 
+  ([ `Integer of int | `Buffer of t | `Uint8Array of Js_typed_array2.Uint8Array.t ] [@bs.unwrap]) ->
+  ?byteOffset:int ->
+  unit -> bool = "includes"
+[@@bs.send]
+
+(** Tests, if [buf] has provided [value] looking from [byteOffset].
+
+    @param buf
+    @param string What to search for.
+    @param byteOffset Where to begin searching in [buf]. If negative, then offset is calculated from the end of [buf]. Default: [0].
+    @param encoding Encoding of the [string]. Default is [`utf8].
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_includes_value_byteoffset_encoding> Node.js API docs
+    @since Node.js v5.3.0
+  *)
+external includesString :
   t -> 
```

Comment:
Looks like we will also need to add a `string` parameter here so it can be called like `includesString t string ()`

Reply:
Sure!

Reply:
Missing argument added.